api/v1alpha1/datadogagent_types.go

```diff
@@ -469,7 +470,7 @@ type Scrubbing struct {
 	Containers *bool `json:"containers,omitempty"`
 }
 
-// PrometheusScrapeConfig allows configuring Prometheus Autodiscovery feature
+// PrometheusScrapeConfig allows configuring Prometheus Autodiscovery feature.
```

Comment:
the wording seems a little awkward; maybe `PrometheusScrapeConfig  allows configuration of the Prometheus Autodiscovery feature.` ?